File: Ruby/day25.rb
def is_prime n
    return false if n == 1
    sqrt_n = Math.sqrt(n).to_i
    for i in 2..sqrt_n
        return false if n % i == 0
    end
    return true
end

n_of_testcase = gets.to_i
for i in 1..n_of_testcase
    n = gets.to_i
    if is_prime n
        puts "Prime"
    else
        puts "Not prime"
    end
end

File: Python/day25.py
import math

def is_prime(n):
    if n == 1:
        return False

    sqrt_n = int(math.sqrt(n))
    for i in range(2, sqrt_n + 1):
        if n % i == 0:
            return False

    return True

n_of_testcase = int(raw_input())
for i in range(n_of_testcase):
    n = int(raw_input())
    if is_prime(n):
        print "Prime"
    else:
        print "Not prime"
